Documentation/botr/xplat-minidump-generation.md

```diff
@@ -57,7 +57,13 @@ Environment variables supported:
 
 (Please refer to MSDN for the meaning of the [minidump enum values](https://msdn.microsoft.com/en-us/library/windows/desktop/ms680519(v=vs.85).aspx) reported above)
 
-**Utility command line options**:
+**Command Line Usage**
+
+The createdump utility can also be run from the command line on arbitrary .NET Core processes. The type of dump can be controlled with the below command switches. The default is a "minidump" which contains the majority the memory and managed state needed. Unless you have ptrace (CAP_SYS_PTRACE) administrative privilege, you need to run with sudo or su. The same as if you were attaching with lldb or other native debugger.
+
+`sudo createdump <pid>`  
+
+**Utility Command Line Options**:
```

Comment:
Nit: Do you need another section header? The command line options seem perfectly reasonable to put in a section called 'Command Line Usage'